python/hsfs/feature_store.py

```diff
@@ -354,6 +355,9 @@ def create_feature_group(
             expectations: Optionally, a list of expectations to be attached to the feature group.
                 The expectations list contains Expectation metadata objects which can be retrieved with
                 the `get_expectation()` and `get_expectations()` functions.
+            event_time: Optionally, provide the name of the feature containing the event
```

Comment:
I think we should have a separate Guide page on the documentation explaining PIT joins.